src/data/value/json.rs

```diff
@@ -0,0 +1,57 @@
+use {
+    super::{Value, ValueError},
+    crate::result::Result,
+    serde_json::Value as JsonValue,
+    std::collections::HashMap,
+};
+
+impl Value {
+    pub fn parse_json_map(value: &str) -> Result<Value> {
+        let value: JsonValue =
+            serde_json::from_str(value).map_err(|_| ValueError::InvalidJsonString)?;
```

Comment:
No problem, but you can exclude the type specification by replacing from_str with [from_value](https://docs.rs/serde_json/1.0.68/serde_json/fn.from_value.html)

Reply:
`JsonValue` here looks unnecessary, which can be obviously inferred by the compiler.
I removed it, thanks :)
However, it does not seem possible to replace `from_str` to `from_value`. It needs to get `&str` and return `JsonValue`.

Reply:
Oh sorry!!
to_value, not from_value.

Reply:
Ignore the above comment
enough for now

Reply:
This is a tiny one, but I also think it would be slightly better to use `serde_json::to_value`.
`serde_json::from_str` clearly shows that the input type is `&str`, but result type can be more than `JsonValue`.
In the case of serde_json::to_value`, it is obvious that the return type is `JsonValue`, but it can accept other input formats.
However, function param clearly shows that the input type of our function is `value: &str`, so.. :)

Reply:
👍 change to match intent

Reply:
Oh, it looks the behavior of `to_value` and `from_str` is different.
serde_json::to_value converts `r#"{ "a": 10 }"#` into `String`, not `Map`.

I'll revert the last commit for now.

Reply:
checked. Wrong change... sorry

Reply:
No problem at all :)
It is very interesting, I'm just quite curious the reason why they have different logics.

Reply:
Naturally, I thought deserialization was necessary.
It would be interesting to know about this.